tests/unit/lms/resources/_js_config_test.py

```diff
@@ -300,7 +303,7 @@ def test_it_sets_the_focused_user_if_theres_a_focused_user_param(
         assert js_config.asdict()["hypothesisClient"]["focus"] == {
             "user": {
                 "username": "example_h_username",
-                "displayName": "example_h_display_name",
+                "displayName": h_api.get_user.return_value.display_name,
```

Comment:
This is nice as it actually tells you why this value is this, rather than just what it is